skimage/__init__.py

```diff
@@ -56,7 +56,6 @@
 
 """
 
-import imp
 import functools
```

Comment:
`functools` import also looks redundant. Am I missing something?

Reply:
@soupault *and* warnings! :joy: `sys` at least is used LOL

Reply:
I believe those were all leftover from a few PRs we were discussing regarding pytest and version checking. I couldn't remove these lines until quite a few mods had been accepted, then I guess I forgot to issue the "cleanup the cleanup" PR.